Sources/MenuAnimator.swift

```diff
@@ -18,8 +18,14 @@
 
 import UIKit
 
+/**
+ Options of menu transitioning.
+ */
```

Comment:
@AndreyArzhannikov what way would you like to see: with a listing of some parameters or without?

1) /**
 Options that define menu transition settings.
 Let user customize their transitioning animation: duration, content scale, using spring or not and other.
 */
2) 1) /**
 Options that define menu transition settings.
 Let user customize their transitioning animation.
 */